third_party/BUILD

```diff
@@ -13,6 +13,7 @@ filegroup(
     name = "all-srcs",
     srcs = [
         ":package-srcs",
+        "//third_party/etcd-v2.2.1-linux-amd64/Documentation/rfc:all-srcs",
```

Comment:
this seems wrong, but I don't know how it got there

Reply:
update bazel did it.

Reply:
To answer my own question: etcd v2.2 included a proto file in the `Documentation/rfc` dir, but it appears to have been removed by etcd v3.

We aren't excluding `third_party/etcd*` from gazelle/kazel, but we probably should be. (We do git ignore these directories.)